docs/multiplatform.md

```diff
@@ -0,0 +1,89 @@
+# Multiplatform
+
+This document covers using kotlin-inject in a multiplatform project.
```

Comment:
looks like this is specifically for android/iOS, can we call that out?

Reply:
Except adding the generated folders to sourcesets, the steps remain same if you add js to the target platform as well. I think I called that out as well.
However, I'm completely fine saying it's mostly towards Android/iOS

Reply:
Thinking something like "below is an example of an android/iOS multiplatform project, other targets follow the same pattern"